hypervisor/start.c

```diff
@@ -402,12 +413,14 @@ void kernelstart(char *arg)
 
     init_gdtidt();
     init_pic();
+    init_pit();
     fifo_init(&key_fifo, 32, keybuffer);
     fifo_init(&mouse_fifo, 128, mousebuffer);
     fifo_init(&serial_fifo, 32, serialbuffer);
     io_sti();
 
-    io_out8(PIC0_DATA, 0xe9); /* PIC0<82>Æ<83>L<81>[<83>{<81>[<83>h<82>ð<8b><96><89>Â(11101001) */
+    io_out8(PIC0_DATA, 0xe8); /* PIC0<82>Æ<83>L<81>[<83>{<81>[<83>h<82>ð<8b><96><89>Â(11101001) */
+    //io_out8(PIC0_DATA, 0xe9); /* PIC0<82>Æ<83>L<81>[<83>{<81>[<83>h<82>ð<8b><96><89>Â(11101001) */
```

Comment:
I would propose not to comment out the code, but to completely remove it. If one wants to back-up he could use git.

Reply:
OK.